README.md

```diff
@@ -91,11 +91,17 @@ var regResult = await f2l.attestationResult(clientAttestationResponse, attestati
 ``` js
 var authnOptions = await f2l.assertionOptions();
 
+// add allowCredentials if you want...
 // save the challenge in the session information...
 // send authnOptions to client and pass them in to `navigator.credentials.get()`...
 // get response back from client (clientAssertionResponse)
 
 var assertionExpectations = {
+     allowCredentials: [{
```

Comment:
I would not add this part to the topmost example, so that copy-pasting is easier to do for checking out the library. I also think its missing a comma. We can either comment it out or just refer to the documentation.

Reply:
I have commented these lines and add some additional comment to explain.